oak/server/rust/oak_loader/src/options.rs

```diff
@@ -100,6 +100,37 @@ impl FromStr for ConfigEntry {
     }
 }
 
+/// A PermissionsConfiguration provides a declarative description of the features that are permitted
+/// for an Oak application.
+///
+/// The permissions are applied dynamically at runtime, when creating the Nodes.
+#[allow(dead_code)]
+#[derive(serde::Deserialize, Debug)]
+#[serde(deny_unknown_fields)]
+pub struct Permissions {
```

Comment:
Are you sure this should be in oak_loader? I was expecting it to be part of oak_runtime? Otherwise we may end up with a circular dep. I may be wrong though.